data/locale/jack_mixer-fr.po

```diff
@@ -0,0 +1,677 @@
+# #-#-#-#-#  jack_mixer-fr.po (jack_mixer 15.1)  #-#-#-#-#
+# jack_mixer i18n message catalog template
+#
+# This file is distributed under the same license as the jack_mixer package.
+#
+# Copyright (C) 2021 Olivier (trebmuh/olinuxx) Humbert <trebmuh@tuxfamily.org>
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: jack_mixer 15.1\n"
+"Report-Msgid-Bugs-To: https://github.com/jack-mixer/jack_mixer/issues\n"
+"POT-Creation-Date: 2021-03-20 14:48+0100\n"
+"PO-Revision-Date: 2021-03-21 18:17+0100\n"
+"Last-Translator: 2021 Olivier (trebmuh/olinuxx) Humbert <trebmuh@tuxfamily.org>\n"
+"Language-Team: French\n"
```

Comment:
Can you set this to:

    "Language-Team: https://github.com/jack-mixer/jack_mixer\n"

, please?

Reply:
I can do that, but that's not what's usual on the others translation I'm working (or have been working) on.
Are you sure?

Please, note that the "Report-Msgid-Bugs-To" field already redirect potential people to the github repo in case of they want to talk about a typo/improvement/mistake...

Reply:
It's not crucial. I thought the field is for contact info of the language team.

Maybe we should put a reference to the `AUTHORS` file and list translators there. For now, just leave it as it is. I can change it later.

Reply:
in .po files, the usual way is to put the translators names and contact [the commented part of the po file](https://github.com/audacity/audacity/blob/master/locale/fr.po#L6).

Reply:
Ok, I checked, and according to https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html the `Language-Team` header field should contain the name of the language of the translation in English and contact address or URL of the language team responsible for the translation.

Since we don't have specific URLs for language teams, and the contact address of individual translators is already given above, I think it's ok leave out the contact info here.

I'll list translator in the `AUTHORS` file nevertheless, just for better visibility. Of course they can still be listed in the comment header as well.